app/policies/food_policy.rb

```diff
@@ -0,0 +1,23 @@
+class FoodPolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  def create?
+    user.present?
+  end
+
+  def update?
+    return true if user.present? && user == food.user
+  end
+
+  def destroy?
+    return true if user.present? && user == food.user
+  end
+
+  private
+
+  def food
```

Comment:
remove, record is fine, because it is food policy and everyone understand which record is.
If you don't think so, let's discuss on a call